x/paychan/codec.proto

```diff
@@ -21,9 +21,13 @@ message PaymentChannel {
   // Total represents a maximum value that can be transferred via this
   // payment channel.
   coin.Coin total = 5;
-  // Absolute block height value. If reached, channel can be closed by
-  // sender.
-  int64 timeout = 6;
+  // Timeout represents wall clock time as read from the block header. Timeout
```

Comment:
This would be an interesting place to try out migrations.

But since there has never been any client code that uses this, not any use in deployed systems, fine to just change it. Waiting for a chance to try real migrations.

Reply:
:grin: I thought about migrations first and then I picked the easy way.

This is the worse migration scenario that we came up. Our migration functions do not have access to the context so there is no nice path for entities migration.